tests/ast/test_folding.py

```diff
@@ -12,7 +12,7 @@ def test_integration():
     test_ast = vy_ast.parse_to_ast("[1+2, 6+7][8-8]")
     expected_ast = vy_ast.parse_to_ast("3")
 
-    folding.fold(test_ast)
+    folding.fold(test_ast, {})
```

Comment:
Does this fail with the changes in this PR?

Regardless, could probably move this to a general cleanup PR